docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -20,8 +20,11 @@ You can use hooks at specific points of select self-service flows:
 - `after` login: The hook runs when the user is successfully authenticated, before the system issues an Ory Session.
 - `before` registration: The hook runs when a registration flow starts.
 - `after` registration: The hook runs when a user successfully registers in the system.
+- `before` recovery: The hook runs when the user starts the recovery flow.
```

Comment:
Taking a step back from this particular tree and looking at the forest:

Is the name of this hook `before` or `before recovery`? If it's the former, this paragraph describes 4 other hooks with the same name. That's confusing. Also, the repetitive phrase ("The hook runs when ...") in each sentence makes this read somewhat clumsy and robotic. What about this shorter and more natural version?

```suggestion
- `before recovery` runs when the user starts the recovery flow.
```

Ideally this also contains a link to the page that explains how to start a recovery flow.

Reply:
This list shows users when they can trigger hooks:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/29116492/188101540-fec99203-ab34-4a92-aae3-605b992b4e09.png">

`before` isn't a hook name, it defines the point at which the hook is triggered.

Reply:
🤔 yeah the list is not very clear. It's not clear how you configure the individual hooks. Ideally, it would probably be linking to somewhere more extensive like the examples below. For this PR, I just wanted to get the new features in and follow the structure already set, but we're always glad and happy to get better documentation so any contribution and improvements are greatly appreciated :))

Reply:
If you look at the list on its own then it doesn't tell you how to configure individual hooks of course. Please look at the complete document and you'll see that this section is meant to list the points at which hooks can be triggered. Examples are provided later on. Everything is easily accessible and scannable using the ToC (right-hand nav. in Docusaurus).

Reply:
Ah, I probably missed that!